Exec/RegTests/CMakeLists.txt

```diff
@@ -38,6 +38,9 @@ endif()
 if(PELE_DIM EQUAL 2)
   add_subdirectory(EB-FlowPastCylinder)
 endif()
+if(PELE_DIM EQUAL 2)
```

Comment:
Another nitpick: just merge this with the if dim=2 conditional immediately above.